src/components/button/button.stories.mdx

```diff
@@ -1,5 +1,16 @@
 import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+// The '!!raw-loader!' syntax is a non-standard, Webpack-specific, syntax.
+// See: https://github.com/webpack-contrib/raw-loader#examples
+// For now, it seems likely Storybook is pretty tied to Webpack, therefore, we are
```

Comment:
https://storybook.js.org/blog/storybook-for-vite/

Storybook has pluggable bundlers now, so it's not very tied to webpack anymore. But we could make a vite plugin for this syntax if we switched to vite

Reply:
I like that idea, @calebeby. I think it's probably a bit bigger than this PR, so I've drafted a new issue. Feel free to edit or comment on it if I've misinterpreted your idea: #1403